arlas-commons/src/main/resources/roles.yaml

```diff
@@ -41,8 +41,10 @@ technicalRoles:
     description:
     - "Download data from ARLAS"
     permissions:
-    - r:/aproc/processes/download/.*:POST
-    - r:/arlas/aproc/processes/download/.*:POST
+    - r:/aproc/processes/download/execution:POST
```

Comment:
what is the difference between 
    - r:/aproc/processes/download/.*:POST
 and
    - r:/arlas/aproc/processes/download/.*:POST
? Why two routes for the same endpoint?